### ISSUE TITLE
Path to down arrow icon for parent menus

### ISSUE THREAD

**Issue by Android63** [NHSLeadership/nightingale#181]
Parent menus should be accompanied by a down-arrow as shown below:

![screen shot 2017-12-12 at 16 59 29](https://user-images.githubusercontent.com/1991226/33897666-fea1fb7e-df5d-11e7-801b-bf473b420cd3.png)

But `.c-nav-primary__link[aria-expanded="false"]:after` uses a url of `/assets/img/sprite/icons.png`, which doesn't work from a theme such as nightingale-wp.

**Comment by csswizardry** [comment]
Do we know why it doesn’t work?

**Comment by penpadstudio** [comment]
@Android63  what happened with this issue?
